src/pages/components/file-uploader/accessibility.mdx

```diff
@@ -19,82 +19,51 @@ import {
-  - [3.2.2 On Input](https://www.ibm.com/able/requirements/requirements/#3_2_2)
-    (WCAG Success Criteria
-    [3.2.2](https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/consistent-behavior-unpredictable-change.html))
-
-## Accessibility testing
-
-Automated, manual and screen reader accessibility verification test has been
-performed on the file uploader React Carbon Component.
-[WCAG 2.1 Level A and AA success criteria](https://www.w3.org/TR/WCAG21/) issues
-have been identified and the list of open accessibility violations is available
-in the Carbon Component GitHub repository.
-
-### Automated test
+## What Carbon provides
+
+Carbon bakes keyboard operation into its components, improving the experience of blind users and others who operate via the keyboard. Carbon incorporates many other accessibility considerations, some of which are described below.
+
+### Keyboard interactions
+
+Both variants of the file uploader provide buttons for uploading and removing files.
+The drop target “Drag and drop files here..." also provides conventional button
+keyboard interaction (`Tab` to reach; `Enter` or `Space` to activate). Once a file
+has been added, it can be removed by activating the delete (“x”) button after each
+file name.
 
 <Row>
-  <Column noGutterSm>
-    <StructuredListWrapper>
-      <StructuredListHead>
-        <StructuredListRow head>
-          <StructuredListCell head>
-            Automated test environment
-          </StructuredListCell>
-          <StructuredListCell head>Results</StructuredListCell>
-        </StructuredListRow>
-      </StructuredListHead>
-      <StructuredListBody>
-        <StructuredListRow>
-          <StructuredListCell>
-            - macOS Mojave version 10.14.2 with VoiceOver
-            <br />
-            - Chrome version 77.0.3865.90
-            <br />
-            - Dynamic Assessment Plugin (DAP) version 1.8.0.0 - IBM
-            Accessibility WCAG 2.1 Sept. 2019 Ruleset
-            <br />- Carbon React version 7.7.1
-          </StructuredListCell>
-          <StructuredListCell>
-            <strong>DAP test:</strong>
-            <br />- No violations
-          </StructuredListCell>
-        </StructuredListRow>
-      </StructuredListBody>
-    </StructuredListWrapper>
-  </Column>
+<Column colLg={8}>
+
+![example of file uploader keyboard interaction](images/file-uploader-accessibility-1.png)
+
+<Caption>The drop zone is also a button that responds to standard keyboard interaction.</Caption>
+
+</Column>
 </Row>
+
+<Row>
+<Column colLg={8}>
+
+![Add file variant with three files attached](images/file-uploader-accessibility-2.png)
+
+<Caption>Uploaded files can be removed by tabbing to each “x” button and activating.</Caption>
+
+</Column>
+</Row>
+
+## Development considerations
+
+Keep these considerations in mind if you’re modifying Carbon or creating a custom component.
+
+* the 'Drag files' area is constructed as a button to support keyboard operation
```

Comment:
I would capitalize the first letter and end with a period for each line here like you have done on other accessibility tab pages.

Reply:
done